src/lib/Workflows.ts

```diff
@@ -171,6 +171,17 @@ export class Workflows {
     }
   }
 
+  private async activateWfs(wfs: IWorkflow[]) {
+    console.log('Activating workflows...');
+    for (const wf of wfs) {
+      try {
+        await this.publicApiClient.workflow.activate(wf.id);
+        console.log(`Activated ${wf.id}.`);
+      } catch (error) {
+        console.error(`Failed to activate ${wf.id}:`, error);
+      }
+    }
+  }
```

Comment:
There is already `activate` method in this class. This one duplicates the existing one logic. Use existing one instead.